timeseries/predict.q

```diff
@@ -5,13 +5,16 @@
 // @kind function
 // @category modelPredict
 // @fileoverview Predictions based on an AutoRegressive model (AR)
-// @param model {dict} Model parameters returned from fitting of an appropriate
-//   model
+// @params config {dict} Information returned from `ml.ts.AR.fit`
+//   including:
```

Comment:
The `including:` instances on can be on the previous line in each of the predict definitions can be on the previous line without hitting the 80 characters